### PR TITLE
Dovecot configuration for docker-mailserver 

### PR DESCRIPTION
**Pull request by ddiawara**
don't know exactly where i past. This is my compose file 

#642

### CODE REVIEW

File: example_configs/mailserver.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+To integrate with LLDAP, make sure you fill in the attributes correctly
+
+Exemple starting [docker-compose](https://github.com/docker-mailserver/docker-mailserver/blob/master/compose.yaml)
+
+Most important part :
+```yaml
+      # >>> Postfix LDAP Integration
+      - ACCOUNT_PROVISIONER=LDAP
+      - LDAP_SERVER_HOST=lldap:3890
+      - LDAP_SEARCH_BASE=dc=d3n,dc=com
```
Comment (nitnelave) [inline example_configs/mailserver.md:14]: dc=example,dc=com

File: example_configs/mailserver.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+      - LDAP_BIND_DN=uid=admin,ou=people,dc=d3n,dc=com
```
Comment (nitnelave) [inline example_configs/mailserver.md:15]: Same here

File: example_configs/mailserver.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+      - LDAP_BIND_PW=admin123$*
+      - LDAP_QUERY_FILTER_USER=(&(objectClass=inetOrgPerson)(|(uid=%u)(mail=%u)))
+      - LDAP_QUERY_FILTER_GROUP=(&(objectClass=groupOfUniqueNames)(|(uid=%s)(cn=%s)))
```
Comment (nitnelave) [inline example_configs/mailserver.md:18]: for LLDAP, uid and cn are the same thing, you can simplify the query to just uid

File: example_configs/mailserver.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+      - LDAP_QUERY_FILTER_ALIAS=(&(objectClass=inetOrgPerson)(|(uid=%u)(mail=%u)))
+      - LDAP_QUERY_FILTER_DOMAIN=(|(mail=*@%s)(mailAlias=*@%s)(mailGroupMember=*@%s))
```
Comment (nitnelave) [inline example_configs/mailserver.md:20]: Here you can remove mail alias and mail group member, they don't exist in LLDAP

File: example_configs/mailserver.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+```shell
+docker network create my_bridge 
+```
+
+## Final Version 
+```yaml
+version: "3.9"
+services:
+  lldap:
+    image: nitnelave/lldap:stable
```
Comment (nitnelave) [inline example_configs/mailserver.md:39]: lldap/lldap

(Same image, but the newer name)

File: example_configs/mailserver.md [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+    ports:
+      - "3890:3890"
+      - "17170:17170"
+    volumes:
+      - "lldap_data:/data"
+    environment:
+      - VERBOSE=true
+      - TZ=Etc/UTC
+      - LLDAP_JWT_SECRET=94721b2ada4bf1ba6462f5eb341ff08372392dbf76
```
Comment (nitnelave) [inline example_configs/mailserver.md:48]: Maybe don't paste your jwt secret? You might want to change it now :)

File: example_configs/mailserver.md [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+      - LLDAP_LDAP_USER_PASS=admin123$*
+      - LLDAP_LDAP_BASE_DN=dc=d3n,dc=com
```
Comment (nitnelave) [inline example_configs/mailserver.md:50]: Dc=example

File: example_configs/mailserver.md [hunk 8]
```diff
@@ -0,0 +1,127 @@
...
+  mailserver:
+    image: ghcr.io/docker-mailserver/docker-mailserver:latest
+    container_name: mailserver
+    hostname: mail.d3n.com
```
Comment (nitnelave) [inline example_configs/mailserver.md:54]: Example

File: example_configs/mailserver.md [hunk 9]
```diff
@@ -0,0 +1,127 @@
...
+      - LOG_LEVEL=debug
+      - SUPERVISOR_LOGLEVEL=debug
+      - SPAMASSASSIN_SPAM_TO_INBOX=1
+      - ENABLE_FAIL2BAN=0
+      - ENABLE_AMAVIS=0
+      - SPOOF_PROTECTION=1
+      - ENABLE_OPENDKIM=0
+      - ENABLE_OPENDMARC=0
+      # >>> Postfix LDAP Integration
+      - ACCOUNT_PROVISIONER=LDAP
```
Comment (nitnelave) [inline example_configs/mailserver.md:82]: Maybe we don't need to repeat the env variables ? Maybe just mention that it's the variables from the top of the post

**Review by nitnelave — COMMENTED** [review]
Thanks! A couple of comments for the form, but lgtm

**Comment by nitnelave** [comment]
Hey @ddiawara , do you still want to work on this?

File: example_configs/mailserver.md (hunk 9 above)
Comment (martadinata666) [reply example_configs/mailserver.md:57]: I removed the top part and adjust at compose sample

**Review by nitnelave — APPROVED** [review]
Sorry, I missed that one! Thanks for updating it

**Pull request merged**